SUMMARY.md

```diff
@@ -99,12 +99,14 @@
         * [Retrieving the Rebalance State](api/rebalance-state.md)
         * [Aborting a Rebalance](api/rebalance-deactivation.md)
     * [Data Retrieval APIs](data-retrieval-apis.md)
-        * [Retrieving Raw Data](api/read-raw.md)
-        * [Retrieving Rollup Data](api/read-rollup.md)
-        * [Retrieving Text Data](api/read-text.md)
-        * [Retrieving Histogram Data](api/read-histogram.md)
+        * [Finding Streams by Name or Tags](api/search-tags.md)
+        * [Retrieving Data](api/fetch.md)
         * [Retrieving Graphite-style Data](api/read-graphite.md)
-        * [Searching Tags](api/search-tags.md)
+        * Legacy Single-Stream Retrieval
+            * [Retrieving Raw Data](api/read-raw.md)
+            * [Retrieving Rollup Data](api/read-rollup.md)
+            * [Retrieving Text Data](api/read-text.md)
+            * [Retrieving Histogram Data](api/read-histogram.md)
```

Comment:
Each of these should get a legacy notice at the top, as was done for the legacy write APIs.